blocklylib-core/src/main/java/com/google/blockly/model/BlockFactory.java

```diff
@@ -72,8 +74,12 @@
     protected final HashMap<BlockTypeFieldName, WeakReference<FieldDropdown.Options>>
             mDropdownOptions = new HashMap<>();
 
+    protected final String mMutatorAltText;
+
     /** Default constructor. */
-    public BlockFactory() {}
+    public BlockFactory(Context context) {
+        mMutatorAltText = context.getResources().getString(R.string.mutator_icon_alt_text);
```

Comment:
All the block definitions need to be reloaded when the language changes as well. Filed #565 to track.